src/arviz_base/io_numpyro.py

```diff
@@ -335,10 +454,23 @@ def from_numpyro(
     coords=None,
     dims=None,
     pred_dims=None,
+    extra_event_dims=None,
     num_chains=1,
 ):
     """Convert NumPyro data into a DataTree object.
 
+    If no dims are provided, this will infer batch dim names from NumPyro model plates.
+    For event dim names, such as with the ZeroSumNormal, `infer={"event_dims":dim_names}`
+    can be provided in numpyro.sample, i.e.::
+        # equivalent to dims entry, {"gamma": ["groups"]}
+        gamma = numpyro.sample(
+            "gamma",
+            dist.ZeroSumNormal(1, event_shape=(n_groups,)),
+            infer={"event_dims":["groups"]}
+        )
+    There is also an additional `extra_event_dims` input to cover any edge cases, for instance
```

Comment:
This seems to be rendering correctly but sphinx is complaining: https://app.readthedocs.org/projects/arviz-base/builds/29253126/#282835299--66. The [official rule](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) is both indented and separated from neighbouring paragraphs by blank lines

```suggestion
    can be provided in numpyro.sample, i.e.::

        # equivalent to dims entry, {"gamma": ["groups"]}
        gamma = numpyro.sample(
            "gamma",
            dist.ZeroSumNormal(1, event_shape=(n_groups,)),
            infer={"event_dims":["groups"]}
        )

    There is also an additional `extra_event_dims` input to cover any edge cases, for instance
```

Reply:
just fixed!